### DISCUSSION TITLE
Error refreshing list of libraries for loading a robot

### DISCUSSION THREAD

**Discussion by Jishnu8** [visose/Robots#213] [Q&A] [answered]
Hi @visose,
I am currently encountering an error when trying to find the list of libraries I can install to load a robot using the "Robots" plugin. Here is a screenshot of the error:

<img width="440" alt="error" src="https://user-images.githubusercontent.com/62787327/215118207-757b287f-8af4-444e-958b-62724f3e3c70.png">

**Comment by visose** [comment]
Did you manually change the content of the that `libraries`? Could you paste a screenshot with the content of that folder?

**Comment by Jishnu8** [comment]
I didn't manually change the contents of the 'libraries' folder. Here is a screenshot of the contents of the folder:

<img width="842" alt="files" src="https://user-images.githubusercontent.com/62787327/215120665-ce758eaa-472f-48ff-ba43-1faddeb46108.png">

However, I do recall that I failed to install the "ETH" library as I got an error. But during that session, I still managed to install a few other libraries after that. I think the error that I mentioned above occurred after that.

**Reply by visose** [reply]
It looks like it downloaded the `ETH.xml` file but not the `ETH.3dm` file for some reason. Try deleting this xml file and check again.

**Comment by Jishnu8** [comment]
You were right! I deleted ETH.xml and EX Architects.xml as they both didn't have the corresponding .3d file. I don't get the error anymore. Thank you!

**Comment by visose** [comment, marked as answer]
Closing this as resolved. The error was caused by orphan/incomplete library files in the local Robots folder, such as an `.xml` without the matching `.3dm`. Delete the broken library files and refresh/reinstall the library.
